docs/what_is_new.md

```diff
@@ -29,7 +29,7 @@
 - 世界状态的存储也从原来的MPT存储结构转为分布式存储，避免了世界状态急剧膨胀导致性能下降的问题。
 - 优化了数据存储的结构，更节约存储空间，存取效率更高。
 
-同时，2.0版本仍然兼容1.0版本的本地存储模式。更多关于存储介绍，请参考[分布式存储设计文档](./design/storage/index.html)
+同时，2.0版本仍然兼容1.0版本的本地存储模式。更多关于存储介绍，请参考[分布式存储设计文档](./design/storage/index.rst)
```

Comment:
For now the link is ok, can you explain why should we modify this?

Reply:
It can't be access when i open this issue, but now it's OK, I'll close it.